corehq/warehouse/transforms/sql/location_dim.sql

```diff
@@ -77,56 +77,3 @@ SET domain = EXCLUDED.domain,
     dim_last_modified = EXCLUDED.dim_last_modified,
     deleted = EXCLUDED.deleted,
     batch_id = EXCLUDED.batch_id;
-
-
--- After the join update all dims with their new foreign keys
```

Comment:
@calellowitz 
I think we should add this back, at least from 0-5 since there are 6 ICDS location levels. Without this, there is no way to get child locations of a location, right?

Reply:
Done here: https://github.com/dimagi/commcare-hq/pull/25390